Source/Resources/Content/AadIam/ExternalUserGuestSettings/Configuration.json

```diff
@@ -1,5 +1,6 @@
 {
   "adminConsentedForUsersIntoTenantIds": [],
+  "description":"What is it: Further defines guests that are allowed to join the tenant by specifying allowed domains. \n Simeon Notes: Allows you to enable specific third parties access into your tenant. This feature is disabled in Simeon's baseline.  Learn More: https://docs.microsoft.com/en-us/azure/active-directory/external-identities/delegate-invitations#to-configure-external-collaboration-settings",
```

Comment:
**What does this do?**
Further defines guests that are allowed to join the tenant by specifying allowed domains. The baseline disables this feature.

**Why should you use this?**
If you do not want to allow entire domain groups access to the tenant, which increases security. 

**What is the end user impact?**
Only specified users, and not those with a specific domain, will be allowed to join the tenant. 

**Learn more**
[Configure B2B external collaboration settings](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/delegate-invitations#to-configure-external-collaboration-settings)

Reply:
@david-simeon  can you see if we are actually changing anything here? If we are, we should probably update the what is it to explain more of what Simeon is doing, not what the feature is.